src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -597,6 +642,21 @@ def _failed_node_supplier(unhealthy_static_nodes: List[SlurmNode], failed_nodes:
                         }
                     }
 
+    def _nodemapping_supplier(self, nodes: List[SlurmNode]) -> Iterator:
```

Comment:
could you separate nodemapping to node_mapping

Reply:
Done